fw/preamp/CMakeLists.txt

```diff
@@ -127,7 +115,7 @@ add_custom_command(TARGET ${PROJECT_NAME}.elf POST_BUILD
 
 add_custom_target(program
   COMMAND cd ${CMAKE_CURRENT_SOURCE_DIR}../../.. &&
-          venv/bin/python -m amplipi.hw --flash ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}.bin
+          python -m amplipi.hw --flash ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}.bin
```

Comment:
hmm

Reply:
What does hmm mean here?

Reply:
an option here that might work well is calling python like so:
```
/usr/bin/env python3
```
this ensures it's python3 that is called, and also tolerates being called from within a virtualenv so you don't have to hardcode any paths.